lb.tf

```diff
@@ -0,0 +1,55 @@
+resource "aws_lb" "main" {
+  count = var.nlb_enabled ? 1 : 0
+
+  name               = var.nlb_name
```

Comment:
Maybe make is so that if var.nlb_name is null then use `${var.broker_name}-nlb"`